noxfile.py

```diff
@@ -146,6 +146,8 @@ def get_sphinx_build_command(kind: str) -> List[str]:
         return [
             "sphinx-build",
             "--keep-going",
+            "--tag",
+            kind,
```

Comment:
Pro tip: it's handy to pass long option values via `=`. This way, you can keep the option coupled with its value, on the same line.
```suggestion
            f"--tag={kind !s}",
```